ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -310,6 +312,261 @@ describe('a4a_config', () => {
       }
     });
   });
+  it('should serve Delayed Fetch to unlaunched DoubleClick filler',
```

Comment:
I initially placed these in test-traffic-experiments.js. However all the tooling/mocking in lines 102-146 are necessary for googleAdsIsA4AEnabled() to function properly so we decided to move it here. But if you disagree with this please let me know and I will be happy to move it.